api/api-doc/storage_service.json

```diff
@@ -104,6 +104,54 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/toppartitions/",
+         "operations":[
+            {
+               "method":"GET",
+               "summary":"Toppartitions query",
+               "type":"toppartitions_query_results",
+               "nickname":"toppartitions_generic",
+               "produces":[
+                  "application/json"
+               ],
+               "parameters":[
+                  {
+                     "name":"name_list",
+                     "description":"Optional list of column family names in keyspace:name format, narrowing down the search. Omitting the name (e.g. 'ks:') will match against all families in the keyspace",
```

Comment:
The description should match the name, column family is the old name for tables, so if the parameter is called cf, the description should talk about column family, you can also change both to table